drivers/led/pca9633.c

```diff
@@ -178,17 +179,17 @@ static inline int pca9633_led_off(const struct device *dev, uint32_t led)
 
 static int pca9633_led_init(const struct device *dev)
 {
+	const struct i2c_dt_spec *config = dev->config;
 	struct pca9633_data *data = dev->data;
 	struct led_data *dev_data = &data->dev_data;
 
-	data->i2c = device_get_binding(DT_INST_BUS_LABEL(0));
-	if (data->i2c == NULL) {
+	if (config->bus == NULL) {
```

Comment:
It should be done this way:

<pre>
if (!device_is_ready(config->bus.bus)) {
	LOG_ERR("I2C bus is not ready");
	return -ENODEV;
}
</pre>